### ISSUE TITLE
Page with a list of key persons and roles is missing

### ISSUE THREAD

**Issue by Lastique** [boostorg/website-v2-docs#184]
There is a page on the current website:

https://www.boost.org/community/moderators.html

Although some names on that page are not the current ML moderators, I think the same kind of page (with the actual list of moderators) should be present on the new website. The page should also describe the duties of a moderator.

Additionally, the page should list not only ML moderators, but also other persons holding key roles in Boost infrastructure, as well as those roles. For example, Release Managers, [Review Wizards](https://www.boost.org/community/reviews.html#Wizard), owners of boostorg on GitHub, etc.

**Comment by joaquintides** [comment]
Do you mean, like retrieving this info from some existing sourec of thruth? I don't think this source exists, but you may ask in Slack to be completely sure.

**Comment by alandefreitas** [comment]
Mmm... I thought each profile on the website was associated with some permissions or tags. At least the moderators must be marked explicitly or they can't moderate anything.

**Comment by rbbeeston** [comment]
With the work that @brianjp93 has done, is this something that can be auto generated Brian?

**Comment by brianjp93** [comment]
@rbbeeston I don't think that any of my work would enable this.  Either a new model would need to be created or if all the moderators have user accounts in boost, maybe an `is_moderator` flag within the User model.  Same for the rest of the key person types.

**Comment by rbbeeston** [comment]
ok, perhaps this would make more sense to look at once mailman3 is implemented.
